boot/freeldr/freeldr/ntldr/winldr.c

```diff
@@ -1267,6 +1261,15 @@ LoadAndBootWindowsCommon(
 
     TRACE("Hello from paged mode, KiSystemStartup %p, LoaderBlockVA %p!\n",
           KiSystemStartup, LoaderBlockVA);
+        
+    /* Copy the page table from paged mode. */
+    if (AcpiTableSource)
+    {
+        /* This is required for 64bit OSes which can throw the BIOS Provided ACPI Tables too high. */
+        RtlCopyMemory(LoaderBlock->Extension->AcpiTable, (PVOID)LoaderBlock, LoaderBlock->Extension->AcpiTableSize);
```

Comment:
You copy the LoaderBlock contents into the AcpiTable?
Shouldn't this be instead:
```suggestion
        RtlCopyMemory(LoaderBlock->Extension->AcpiTable, (PVOID)AcpiTableSource, LoaderBlock->Extension->AcpiTableSize);
```

Reply:
Oop that’s a commit mess up